lib/modules/manager/maven/extract.ts

```diff
@@ -76,6 +84,84 @@ function containsPlaceholder(str: string | null | undefined): boolean {
   return !!str && regEx(/\${[^}]*?}/).test(str);
 }
 
+function getCNBDependencies(
+  nodes: XmlElement[],
+  config: ExtractConfig,
+): PackageDependency[] {
+  const deps: PackageDependency[] = [];
+  nodes.forEach((node) => {
+    const depString = node.val.trim();
+    if (isDockerRef(depString)) {
+      const dep = getDockerDep(
+        depString.replace(DOCKER_PREFIX, ''),
+        true,
+        config.registryAliases,
+      );
+      dep.registryUrls = [];
```

Comment:
why is this needed here? `buildpacks/extract` doesn't set it either?

Reply:
if dep.registryUrls is not set, extract fails here: https://github.com/renovatebot/renovate/blob/c1d59fde1d831c6c8833c4d9774731fd7a9c55eb/lib/modules/manager/maven/extract.ts#L585

Reply:
Done